identity-service/src/utils/relayHelpers.js

```diff
@@ -12,6 +31,37 @@ const allowedProgramIds = new Set([
   /* secp */ 'KeccakSecp256k11111111111111111111111111111'
 ])
 
+/**
+ * Maps the instruction enum to the index of the rewards manager authority account
+ * @see {@link [../../../solana-programs/reward-manager/program/src/instruction.rs](https://github.com/AudiusProject/audius-protocol/blob/db31fe03f2c8cff357379b84130539d51ccca213/solana-programs/reward-manager/program/src/instruction.rs#L60)}
+ */
+const rewardManagerAuthorityIndices = {
+  0: 4, // InitRewardManager
+  1: -1, // ChangeManagerAccount
```

Comment:
`ChangeManagerAccount` and `DeleteSenderPublic` don't take in the authority account. Should I be using "RewardManager" instead?

Reply:
Yeah... I think that is actually better